services/ui/src/components/ProjectTeaser/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Link from 'next/link';
+
+export default ({ project }) => (
+  <tr>
+    <td>
+      <Link href={{ pathname: '/project', query: { name: project.name } }}>
+        <a>{project.name}</a>
+      </Link>
+    </td>
+    <td>{project.customer.name}</td>
+  </tr>
+);
```

Comment:
This doesn't feel like a reusable component to me. Will project teasers always be in a table? It's not very complex, could just as easily be inlined in pages/index.js